docs/DeveloperGuide.md

```diff
@@ -410,6 +504,40 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   * 2a1. TAHub shows an error message stating that the index is invalid.
   * 2a2. Use case ends.
 
+**<u>Use case: UC6 - Create a Consultation</u>**
+
+**MSS**
+
+1. User enters: `addconsult d/2024-10-20 t/14:00`
```

Comment:
Do we also need to check if the date and time are before or after the current date and time? i.e. adding a consultation for May when it is already June.

Reply:
> Do we also need to check if the date and time are before or after the current date and time? i.e. adding a consultation for May when it is already June.

I don't think we need to add a hard restriction for this. Maybe the TA wants to do it to record that there was a consultation that took place earlier or something. Instead, maybe the GUI can show past consultations with their dates highlighted in red?